packages/astro-purgecss/src/utils.ts

```diff
@@ -18,7 +18,6 @@ export async function writeFileContent(filePath: string, content: string) {
     await writeFile(filePath, content, 'utf8');
   } catch (err) {
     error(`Error writing file ${filePath}: ${err}`);
-    return '';
```

Comment:
I noticed the function calls don't use the return value and the first branch doesn't return any value anyways.